hudi-common/src/main/java/org/apache/hudi/common/config/HoodieCommonConfig.java

```diff
@@ -136,6 +136,19 @@ public class HoodieCommonConfig extends HoodieConfig {
       .markAdvanced()
       .withDocumentation("Property to control the max memory in bytes for dfs input stream buffer size");
 
+  public static final ConfigProperty<Boolean> HOODIE_FILE_INDEX_USE_SPILLABLE_MAP = ConfigProperty
+      .key("hoodie.file.index.cache.use.spillable.map")
+      .defaultValue(false)
```

Comment:
Could we turn this on by default as it's validated by production workload?

Reply:
It's not completely rolled out internally and enabled for orgs where driver memory is low, still better to keep it false for OSS.

Reply:
Got it.